contracts/common/lib/MerklePatriciaProof.sol

```diff
@@ -42,8 +42,9 @@ library MerklePatriciaProof {
         return false;
       }
 
-      currentNode = RLPReader.toBytes(parentNodes[i]);
+      currentNode = RLPReader.toRlpBytes(parentNodes[i]);
```

Comment:
RLPReader issue identified by trial and error.

Reply:
haha :d